jerry-core/jmem/jmem-heap.c

```diff
@@ -124,6 +124,7 @@ jmem_heap_init (void)
 
 #ifndef JERRY_SYSTEM_ALLOCATOR
   JERRY_ASSERT ((uintptr_t) JERRY_HEAP_CONTEXT (area) % JMEM_ALIGNMENT == 0);
```

Comment:
The `JERRY_HEAP_CONTEXT` uses the current context macro internally. Maybe in a follow up patch that can be also simplified to use the new INIT macro.